city_scrapers/spiders/cook_emergency_telephone.py

```diff
@@ -0,0 +1,135 @@
+import re
+from datetime import datetime, timedelta
+from io import BytesIO
+
+import scrapy
+from city_scrapers_core.constants import FORUM
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from PyPDF2 import PdfFileReader
+
+
+class CookEmergencyTelephoneSpider(CityScrapersSpider):
+    name = "cook_emergency_telephone"
+    agency = "Cook County Emergency Telephone System Board"
+    timezone = "America/Chicago"
+    start_urls = ["https://www.cookcounty911.com/"]
+    location = {
+        "name": "Conference Room",
+        "address": "1401 S. Maybrook Drive, Maywood, IL 60153",
+    }
+
+    def __init__(self, *args, **kwargs):
+        self.meeting_starts = []
+        self.docs_link = ""
+        self.agenda_link = ""
+        super().__init__(*args, **kwargs)
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        schedule_link = ""
+        for link in response.css(".sliding_box a"):
+            link_text = link.css("*::text").get()
+
+            schedule_link = self.start_urls[0] + '/wp-content/uploads/pdfs/'
+
+            if "Minutes" in link_text:
+                self.docs_link = link.attrib["href"]
+            elif "Agenda" in link_text:
+                self.agenda_link = link.attrib["href"]
+
+        if schedule_link and self.docs_link:
+            yield scrapy.Request(
+                response.urljoin(schedule_link), callback=self._parse_schedule, dont_filter=True
+            )
+        else:
+            raise ValueError("Required links not found")
+
+    def _parse_schedule(self, response):
+        """Parse PDF and then yield to documents page"""
+        schedule_pdf_link = 'schedule.pdf'
```

Comment:
Since we're only using this variable once I think it'll be more straightforward to remove it and just request `response.urljoin('schedule.pdf')`